core/templates/vector.h

```diff
@@ -69,7 +69,11 @@ class Vector {
 	bool push_back(T p_elem);
 	_FORCE_INLINE_ bool append(const T &p_elem) { return push_back(p_elem); } //alias
 	void fill(T p_elem);
-
+	void remove_back() {
+		if (_cowdata.size() > 0) {
+			remove_at(_cowdata.size() - 1);
```

Comment:
```suggestion
			_cowdata.resize(_cowdata.size() - 1);
```
Using `remove_at` like this is less expensive